pyccel/codegen/printing/ccode.py

```diff
@@ -668,6 +698,12 @@ def _print_FunctionDef(self, expr):
         decs += [Declare(i.dtype, i) for i in self._additional_declare]
         decs  = '\n'.join(self._print(i) for i in decs)
         self._additional_declare.clear()
+
+        if len(expr.results) > 1:
+            self._additional_args = [a.clone(name = self._parser.get_new_name('tmp_ret'), is_pointer =True) for a in expr.results]
+            body += '\n'.join(self._print_Assign(Assign(a, b)) for  a, b in zip(self._additional_args, expr.results))
+            body += '\n' + self._print_Return(Return([1]))
```

Comment:
Why do you always return 1?

Reply:
what I mean by that is that function with multiple results is always true 
```
def func():
    return 0, 0
bool(func())  ===> True
```

Reply:
And maybe I need to make it return boolean better

Reply:
Hmm, personally I am not a fan of doing return True when it cannot be False, but it can sometimes be useful so I am fine with you leaving it.
But if you want to do this then I would either:
1. return True (bools are implemented in c, why not use them?)
2. return 0 (integer code for no error)